modules/veteran/app/controllers/veteran/v0/base_accredited_representatives_controller.rb

```diff
@@ -9,10 +9,12 @@ class BaseAccreditedRepresentativesController < ApplicationController
       before_action :verify_sort
       before_action :verify_long
       before_action :verify_lat
+      before_action :verify_distance
 
       DEFAULT_PAGE = 1
       DEFAULT_PER_PAGE = 10
       DEFAULT_SORT = 'distance_asc'
+      PERMITTED_MAX_DISTANCES = [5, 10, 25, 50, 100, 200].freeze # in miles, no distance provided will default to "all"
```

Comment:
From Figma:
![image](https://github.com/department-of-veterans-affairs/vets-api/assets/144135615/88cbac00-de7e-4037-a753-ca5322941ec6)

Reply:
I thought about including the distance options listed on the FE explicitly but I didn't think that was necessary. I'm fine with this.